services/src/main/resources/org/keycloak/protocol/oidc/endpoints/login-status-iframe.html

```diff
@@ -72,6 +72,10 @@
 
         // If the client and origin from the event match the verified ones from the server, signal if the cookie has changed.
         if (clientId === init.clientId && origin === init.origin) {
+          const hashedSessionId = await hashString(sessionState);
+          if (hashedSessionId === cookie) return "unchanged";
+
+          // Backwards compatibility with versions older than 26.1
```

Comment:
Same for this code, perhaps we log an issue to remove this in the next major.

Reply:
Logged https://github.com/keycloak/keycloak/issues/34772